app/controllers/concerns/idv_step_concern.rb

```diff
@@ -12,8 +12,13 @@ def confirm_document_capture_complete
     @pii = flow_session&.[]('pii_from_doc') # hash with indifferent access
     return if @pii.present?
 
-    flow_session&.delete('Idv::Steps::DocumentCaptureStep')
-    redirect_to idv_doc_auth_url
+    if (IdentityConfig.store.doc_auth_document_capture_controller_enabled &&
```

Comment:
Could we pull out `flow_path` as a variable like we did in the other controller to shorten the line? Also drop the outer parens